src/modules/users-login-routes/index.js

```diff
@@ -34,32 +37,32 @@ export default class UsersLoginRoutes extends HasUserModel {
     var email = req.param('email')
     return this.models.User.findOne({email}).then((user) => {
       if(!user) throw new Error('No user matching that email was found.')
-      var link = "http://"+this.app.config.baseUrl+"/login-link?token="+user.resetPasswordToken
+      var link = "http://"+this.app.config.baseUrl+baseUrl+"login-link?token="+user.resetPasswordToken
```

Comment:
I don't see baseUrl defined in this scope

And not this PR's fault, but this.app is also an error now.